### PR TITLE
docs(sdk): refresh docs to reflect latest changes in sdk.

### PR DESCRIPTION
**Pull request by vkubiv**
Signed-off-by: Volodymyr Kubiv <volodymyr.kubiv@euristiq.com>

### CODE REVIEW

**Review by talwinder50 — APPROVED** [review]

File: cmd/wallet-sdk-gomobile/docs/usage.md [hunk 1]
```diff
@@ -76,8 +77,9 @@ then an in-memory key store will be used.
 
 ```kotlin
 import dev.trustbloc.wallet.sdk.localkms
```
Comment (DRK3) [inline cmd/wallet-sdk-gomobile/docs/usage.md:79]: Is `import dev.trustbloc.wallet.sdk.localkms` still needed?

File: cmd/wallet-sdk-gomobile/docs/usage.md [hunk 2]
```diff
@@ -383,28 +391,33 @@ They use in-memory key storage and the Tink crypto library.
...
+val documentLoader = DocLoader()
+val didDocResolution = didCreator.create("key", CreateDIDOpts()) // Create a did:key doc
 
 // Going through the flow
 val interaction = openid4vp.Interaction("YourAuthRequestURIHere", kms, kms.getCrypto(), didResolver, docLoader)
 val query = interaction.getQuery()
 val inquirer = credential.Inquirer(docLoader)
 val issuedCredentials = api.VerifiableCredentialsArray() // Would need some actual credentials for this to actually work
 val verifiablePres = inquirer.Query(query, credential.CredentialsOpt(issuedCredentials))
+val matchedCreds = verifiablePresentation.credentials() // this is to show on comfimation dialog.
```
Comment (DRK3) [inline cmd/wallet-sdk-gomobile/docs/usage.md:418]: Typo in `comfimation` - should say `confirmation`.

Also, is this message specific to our reference app? If so, I don't think it belongs here as this is general-purpose documentation. Perhaps you can word it in a more general way.

**Review by DRK3 — CHANGES_REQUESTED** [review]
Please see my comments

File: cmd/wallet-sdk-gomobile/docs/usage.md (hunk 1 above)
Comment (DRK3) [reply cmd/wallet-sdk-gomobile/docs/usage.md:79]: Disregard this

File: cmd/wallet-sdk-gomobile/docs/usage.md (hunk 2 above)
Comment (DRK3) [reply cmd/wallet-sdk-gomobile/docs/usage.md]: I would recommend changing the wording to something like "// This should be shown to the user with a confirmation dialog so they can confirm that they want to share this data before calling presentCredential` to make the intent more clear to the user of the SDK.

**Review by DRK3 — APPROVED** [review]
Looks good - thanks so much for updating the docs!!

**Pull request merged**